evm2wast.h

```diff
@@ -0,0 +1,205 @@
+#ifndef __EVM2WAST_H
+#define __EVM2WAST_H
+#include <stdlib.h>
+#include "gadgets.h"
+
+struct opcode {
+	char *name;
+	size_t fee;
+	int off_stack;
+	int on_stack;
+};
+
+enum op_nums {
+	STOP = 0x00,
+	ADD, MUL, SUB, DIV, SDIV, MOD, SMOD,
+	ADDMOD, MULMOD, EXP, SIGNEXTEND,
+	LT = 0X10, GT, SLT, SGT, EQ, ISZERO,
```

Comment:
Can you keep using lowercase `x` in the hex values?

Reply:
typo